### DISCUSSION TITLE
How to start an activity and get the result

### DISCUSSION THREAD

**Discussion by ExtSolTech** [dotnet/maui#8465] [Q&A]
Hy All,

I want to prompt the user to select one account and i will get the email address but i cant start an activity.

Here is some resources:
https://developers.google.com/android/reference/com/google/android/gms/common/AccountPicker
https://www.mongodb.com/developer/languages/kotlin/realm-startactivityforresult-registerforactivityresult-deprecated-android-kotlin/
https://docs.microsoft.com/en-us/dotnet/maui/fundamentals/app-lifecycle

so, i add an event handler to catch the OnActivityResult in static class MauiProgram:
```
builder.ConfigureLifecycleEvents(e => 
{
    e.AddAndroid(android => android.OnActivityResult((act, req, res, intent) => 
    {
                    
     }));
});
```

and try to execute another activity in viewmodel on button click occurred:
```
[RelayCommand]
internal void Pick()
{
    Intent intent =AccountPicker.NewChooseAccountIntent(
            new AccountPicker.AccountChooserOptions.Builder().SetAllowableAccountsTypes(
                new[] { "com.google" }).Build());

    startActivityForResult(intent, 10); // ???
}
```

but the `startActivityForResult` method is not exists, or i not found which namespace is contain.
What is the equivalent `startActivityForResult` in MAUI?

ty

Best regards,
ExtSol

**Comment by ZahidMehmoodquarterpi** [comment]
You can start Activity for result in .Net Maui like that:
```csharp
Microsoft.Maui.ApplicationModel.Platform.CurrentActivity.StartActivityForResult(Intent, 1);
```

**Reply by cunj123** [reply]
Should this still work?
